pkg/inject/inject.go

```diff
@@ -757,6 +759,20 @@ func (conf *ResourceConfig) tapDisabled() bool {
 	return false
 }
 
+func (conf *ResourceConfig) proxyWaitBeforeExitSeconds() (uint, error) {
+	if override := conf.getOverride(k8s.ProxyWaitBeforeExitAnnotation); override != "" {
+		waitBeforeExitSeconds, err := strconv.ParseUint(override, 10, 32)
```

Comment:
If we use `uint64`, then we can simplify this to
```
return strcconv.ParseUnit(override, 10, 64)
```

Reply:
But who will ever need the sleeping period longer than 136,19251950152 years?

Reply:
I am hoping we can explore ways to simplify the code, without needing all the type casting. The limit of the type isn't a concern here.

Reply:
Done!